charts/hello/templates/tests/test-connection.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
 {{ include "hello.labels" . | nindent 4 }}
   annotations:
     "helm.sh/hook": test-success
+    policy.otomi.io/ignore: psp-allowed-users
```

Comment:
why do you hardcode it here, and use `podAnnotations` everywhere else? Stay consistent and use `podAnnotations` only. It has the advantage of managing them from the values folder only.